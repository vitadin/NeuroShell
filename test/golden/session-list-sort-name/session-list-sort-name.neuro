# Test session list with sort by name
# Create sessions in reverse alphabetical order, then sort by name

# Create sessions in reverse alphabetical order
\session-new zebra_session
\session-new alpha_session
\session-new beta_session

# List sessions sorted alphabetically by name
\session-list[sort=name]