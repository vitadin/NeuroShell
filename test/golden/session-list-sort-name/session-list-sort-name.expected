Setting _echo_commands = true
%%> \session-new zebra_session
Created session 'zebra_session' (ID: 00000001)
%%> \session-new alpha_session
Created session 'alpha_session' (ID: 00000002)
%%> \session-new beta_session
Created session 'beta_session' (ID: 00000003)
%%> \session-list[sort=name]
Sessions (3 total):
  alpha_session    (ID: 00000002, 0 messages, created: 2025-01-01)
  beta_session    (ID: 00000003, active, 0 messages, created: 2025-01-01)
  zebra_session    (ID: 00000001, 0 messages, created: 2025-01-01)